docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -324,25 +394,37 @@ Maps keys to translation files, based on the locale indicated by the visitor’s
 
 [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/lang.js).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=lang).
-
 ### {{langJson}}

```

Comment:
@junedkazi Can you please provide an example for this helper?

Reply:
We have a couple of examples in cornerstone https://github.com/bigcommerce/cornerstone/search?q=langJson
Lets take this one for example https://github.com/bigcommerce/cornerstone/blob/0c2a8824e79f7bb9a52111558330f70d9a809593/templates/layout/base.html#L56

It will basically load the following messages in json formant for the storefront locale/language https://github.com/bigcommerce/cornerstone/blob/851e42b1f599a711cdafc5c3cf616ce2719edc75/lang/en.json#L912